src/api/controllers/lines.js

```diff
@@ -171,5 +202,5 @@ function updateLineStatus(req, res) {
     }
 }
 
-export default { getLineByAuthUser, getLineById, createLine, updateLine, deleteLine, getStatsByEmployerId, updateLineStatus }
+export default { getLineByAuthUser, getLineById, createLine, updateLine, deleteLine, getStatsByEmployerId, getUsersByEmployerId,updateLineStatus }
```

Comment:
`+export default { getLineByAuthUser, getLineById, createLine, updateLine, deleteLine, getStatsByEmployerId, getUsersByEmployerId,updateLineStatus }`

add space between getUsersByEmployerId and updateLineStatus